scripts/scraper-ng/test/ingredient-data.examples.test.js

```diff
@@ -0,0 +1,107 @@
+const { process } = require("./framework/utils");
+
+const sources = {
+  valid_single_simple: `<h2 id="Examples">Examples</h2>
+<h3 id="An_example">An examples</h3>`,
+
+  valid_multiple_simple: `<h2 id="Examples">Examples</h2>
+<h3 id="An_example">An example</h3>
+<h3 id="Another_example">Another example</h3>`,
+
+  valid_multiple_live_samples: `<h2 id="Examples">Examples</h2>
+<h3 id="An_example">An example</h3>
+  <p>Some prose description</p>
+  <pre class="brush: html">&lt;h1&gt;A heading&lt;/h1&gt;</pre>
+  <pre class="brush: css">h1: { color: blue; }</pre>
+  {{EmbedLiveSample("An_example")}}
+<h3 id="Another_example">Another example</h3>
+  <p>Some more prose description</p>
+  <pre class="brush: html">&lt;h1&gt;A heading&lt;/h1&gt;</pre>
+  <pre class="brush: css">h1: { color: red; }</pre>
+  {{EmbedLiveSample("Another_example")}}`,
+
+  valid_simple_live_mixture: `<h2 id="Examples">Examples</h2>
+<h3 id="An_example">An example</h3>
+  <p>Some prose description</p>
+  <pre class="brush: html">&lt;h1&gt;A heading&lt;/h1&gt;</pre>
+  <pre class="brush: css">h1: { color: blue; }</pre>
+  {{EmbedLiveSample("An_example")}}
```

Comment:
Maybe we could have some newlines after this macro call, since the spec mentions this as a possibility

```suggestion
  {{EmbedLiveSample("An_example")}}
  
  
```

Reply:
Oh, I somehow didn't see the suggested change, and just added this independently.